.circleci/config.yml

```diff
@@ -6,25 +6,26 @@ jobs:
     steps:
       - checkout
       - restore_cache:
-          key: dependency-cache-{{ checksum "packages/cf.js/package.json" }}{{ checksum "packages/contracts/package.json"}}{{ checksum "packages/iframe-wallet-poc/package.json"}}{{ checksum "packages/machine/package.json" }}{{  checksum "packages/playground/package.json" }}{{ checksum "packages/typescript-typings/package.json" }}
```

Comment:
why change the key?

Reply:
The PR introduces a new package so the key needs to be updated.